ale_linters/java/checkstyle.vim

```diff
@@ -39,11 +39,17 @@ function! ale_linters#java#checkstyle#Handle(buffer, lines) abort
     return l:output
 endfunction
 
+function! s:GetConfig(buffer, config) abort
+    let s:file = ale#path#FindNearestFile(a:buffer, a:config)
```

Comment:
We can probably avoid checking the filesystem at all if the configuration file starts with `/`.

Reply:
Indeed.